File: test/fixtures/java-jsoup/get_with_data.java
import java.io.IOException;
import java.io.InputStream;
import java.net.HttpURLConnection;
import java.net.URL;
import java.util.Scanner;

class Main {

	public static void main(String[] args) throws IOException {

		Connection.Response response = Jsoup.connect("http://localhost:28139/synthetics/api/v3/monitors?test=2&limit=100&w=4")
				.header("X-Api-Key", "123456789")
				.method(Connection.Method.GET)
				.ignoreContentType(true)
				.timeout(30000)
				.execute();

		System.out.println(response.parse());
	}
}

File: test/fixtures/javascript-jquery/get_with_data.js
$.ajax({
  url: 'http://localhost:28139/synthetics/api/v3/monitors',
  crossDomain: true,
  headers: {
    'X-Api-Key': '123456789'
  },
  data: {
    'test': '2',
    'limit': '100',
    'w': '4'
  }
}).done(function(response) {
  console.log(response);
});
